docs/integrations/monitoring/datadog.md

```diff
@@ -2,31 +2,115 @@
 
 You can add operational visibility to your [Rack](../../reference/primitives/rack) with Datadog.
 
+## Configure Kubectl to Point at Your Rack
+
+Convox allows you to securely connect your Kubectl to your Convox created Kubernetes cluster by exporting a [Kubeconfig](../../reference/cli/rack#rack-kubeconfig) that will connect you to a Kubernetes API Proxy running inside your Rack. This allows you to use Kubectl without directly exposing the credentials for your Kubernetes cluster. For example if your Rack is called `myrack` you could point your local Kubectl to your Rack cluster as follows
+
+```
+$ convox rack kubeconfig /tmp/myrack-config
+$ export KUBECONTROL=/tmp/myrack-config
+```
+
+This will export the proxy configuration to a temporary file and then point your local Kubecontrol at that location so you can connect to your Rack's cluster. You will need to perform this step before you can execute any Kubectl commands against your cluster.
```

Comment:
```suggestion
This will export the proxy configuration to a temporary file and then point your local Kubectl environment at that location so you can connect to your Rack's cluster. You will need to perform this step before you can execute any Kubectl commands against your cluster.
```